packages/api/indexer/processor/hustler.go

```diff
@@ -237,6 +294,30 @@ func (p *HustlerProcessor) ProcessMetadataUpdate(
 
 	safeName := CleanJsonString(meta.Name)
 
+	// LOG BEFORE UPDATING …and deal with stupid nil pointers
+	var beardIDStr string
+	if (beardID) == nil {
+		beardIDStr = ""
```

Comment:
You can remove this case since beardIDStr should already be an empty string because of the var string

Reply:
That's not how Go strings work with nils as far as I understand with pointers. Unless I guard with this clause it triggers a nil pointer reference.

Reply:
Good catch thanks it was late!